crates/dap_adapters/src/python.rs

```diff
@@ -572,12 +636,17 @@ impl DebugAdapter for PythonDebugAdapter {
             if let Some(path) = Path::new(&toolchain.path.to_string()).parent() {
                 let debugpy_path = path.join("debugpy");
                 if smol::fs::metadata(&debugpy_path).await.is_ok() {
```

Comment:
```suggestion
                if delegate.fs().is_file(&debugpy_path).await {
```

We have a `Fs` trait that is usually accessible from `AppState`, which is on `Workspace` among other types. 

If you need to do file system operations, it's a good idea to see if the `Fs` trait has a method that does what you need. It's more idiomatic and more readable (in this case).

Reply:
done 👍